message-service/src/main/java/sjchat/messages/MessageService.java

```diff
@@ -1,11 +1,33 @@
 package sjchat.messages;
 
+import java.util.List;
 import java.util.Random;
 
+import io.grpc.Channel;
+import io.grpc.ManagedChannelBuilder;
 import io.grpc.stub.StreamObserver;
+import sjchat.daos.ChatDao;
+import sjchat.daos.ChatDaoImpl;
+import sjchat.daos.MessageDao;
+import sjchat.daos.MessageDaoImpl;
+import sjchat.entities.ChatEntity;
+import sjchat.entities.MessageEntity;
+import sjchat.exceptions.NoEntityExistsException;
+import sjchat.users.GetUserRequest;
 import sjchat.users.User;
+import sjchat.users.UserServiceGrpc;
 
 class MessageService extends MessageServiceGrpc.MessageServiceImplBase {
+  MessageDao messageDao = new MessageDaoImpl();
+  ChatDao chatDao = new ChatDaoImpl();
+  private Channel userServiceChannel = buildUserServiceChannel();
+
+  private static Channel buildUserServiceChannel() {
+    String host = System.getenv("USER_SERVICE_HOST");
+    host = (host == null) ? "localhost" : host;
+
+    return ManagedChannelBuilder.forAddress(host, 50051).usePlaintext(true).build(); //TODO: Put port in config file
```

Comment:
Remove the TODO comment. Please create issues instead of //TODO comments so we have all issues / todos in one place.

I created an issue at #80